### PR TITLE
Make geocat work with https with protocol relative links

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: web-ui/src/main/resources/catalog/components/common/map/print/PrintDirective.js [hunk 1]
```diff
@@ -494,7 +494,7 @@
             var tileGrid = source.getTileGrid();
             angular.extend(enc, {
               type: 'WMTS',
-              baseURL: location.protocol + '//wmts.geo.admin.ch', // FIXME
+              baseURL: '//wmts.geo.admin.ch', // FIXME
```
Comment (pvalsecc) [inline web-ui/src/main/resources/catalog/components/common/map/print/PrintDirective.js]: Maybe remove the FIXME...

**Comment by pvalsecc** [comment]
LGTM

File: web-ui/src/main/resources/catalog/components/common/map/print/PrintDirective.js (hunk 1 above)
Comment (Vampouille) [inline web-ui/src/main/resources/catalog/components/common/map/print/PrintDirective.js]: I don't known what has to be fixed in this line. I think it is more related to the hardcoded domain than protocol.

**Comment by Vampouille** [comment]
With this PR, this should work with HTTPS : no more mixed content alert from browser.

In order to work with HTTPS, changes need to be done in Settings/Catalog server :
* Preferred Protocol : https
* **Port: https port : 443 or 8493 for dev**
* Secure Port : same as above

**Comment by pvalsecc** [comment]
Still LGTM, please merge, then ping me (I'll do a deploy on int).

**Pull request merged**